src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2025,6 +2025,9 @@
        from expression can occur. -->
   <Node Name="BoundInterpolatedStringArgumentPlaceholder" Base="BoundValuePlaceholderBase">
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow" />
+    <!-- The index in the containing member of the argument this is the placeholder for. Should be a positive number or
+         one of the constants in the other part of the partial class. -->
```

Comment:
I think `BoundInterpolatedStringArgumentPlaceholder` doesn't survive lowering. If so, we can add a comment (see `BoundDeconstructValuePlaceholder` for example) and enforce it in `LocalRewritingValidator`.